modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/java/com/liferay/osb/faro/service/impl/FaroUserLocalServiceImpl.java

```diff
@@ -165,6 +165,24 @@ public void deleteFaroUsers(long groupId) {
 		faroUserPersistence.removeByGroupId(groupId);
 	}
 
+	@Override
+	public void deleteFaroUsers(User user) throws PortalException {
+		List<FaroUser> faroUsers = new ArrayList<>();
+
+		for (FaroProject project : _faroProjectPersistence.findAll()) {
```

Comment:
@sillascavalcanti , please create add a new finder by LiveUserId in the service.xml instead of iterating thu all projects. 

```
<finder name="LiveUserId" return-type="Collection"> 
<finder-column name="liveUserId" /> 
</finder> 
```